homeassistant/components/rfxtrx/__init__.py

```diff
@@ -203,16 +212,20 @@ def unload_internal(hass, config):
     rfx_object.close_connection()
 
 
-def setup_internal(hass, config):
+def setup_internal(hass, entry: config_entries.ConfigEntry):
     """Set up the RFXtrx component."""
+    config = entry.data
+
     # Setup some per device config
+    devices = set()
     device_events = set()
     device_bits = {}
```

Comment:
Now that there exist a config entry even for discovered devices, I think we can drop these lookups and just keep an dict with the device_id as key and the config data as values.

That way we don't need these separate lookups. We can do it as a refactor change later though.

Reply:
Would that work efficiently? device_id is still a tuple.

Reply:
Never mind that.